circle-mlir/infra/docker/u2004/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update \
 
 # Build tool
 RUN apt-get update \
- && apt-get -qqy install build-essential cmake git fakeroot autoconf automake libtool unzip wget \
+ && apt-get -qqy install build-essential git fakeroot autoconf automake libtool unzip wget \
```

Comment:
as we install cmake 3.22 from cmake.org, we don't need to install Debian package.